src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -80,8 +80,16 @@ export default function schoolSearch(state = initialState, action) {
 
       const institutionQuery = action.institutionQuery;
       const institutions = data.map(({ attributes }) => {
-        const { city, country, facilityCode, name, state: institutionState, zip } = attributes;
-        return { city, country, facilityCode, name, state: institutionState, zip };
+        const { city, country, facilityCode, name, state: institutionState, street, zip } = attributes;
+        return {
```

Comment:
I want to be able to inline some conditional logic in the `jsx` for the appearance of these strings. String templates will render `false` and `null` but not an empty string 🙄 and the service returns `null` when an attribute is not available.

Reply:
How about using lodash reduce to build this?

Reply:
Maybe- it's using the native array map to transform an array of objects. What would lodash's reduce help with?

Reply:
line 84 could be refactored as something like:
```
return _.reduce(attributes, (result, value, key) => {
  result[key] = value || '';
  return result;
}, {});
```